src/parser/expression.js

```diff
@@ -316,7 +316,7 @@ pp.parseSubscripts = function (base, startPos, startLoc, noCalls) {
     } else if (this.match(tt.backQuote)) {
       let node = this.startNodeAt(startPos, startLoc);
       node.tag = base;
-      node.quasi = this.parseTemplate();
+      node.quasi = this.parseTemplate(true);
```

Comment:
I'm not a fan of passing boolean as argument in a function. I think it reduce readability.

I would suggest creating two *separated* functions: `parseTaggedTemplate` and `parseTemplate`. What do you think about that?